enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -49,10 +48,11 @@ const SdkContentWrapperInner = styled.div<
   --mb-color-text-medium: ${({ theme }) => theme.fn.themeColor("text-medium")};
   --mb-color-text-light: ${({ theme }) => theme.fn.themeColor("text-light")};
 
+  font-size: ${({ theme }) => theme.other.fontSize ?? "0.875em"};
```

Comment:
we can lift `getWrapperStyle` here as `theme.other.fontSize` exists in the global Mantine scope now